src/components/Form/Inputs/SearchSelect/SearchSelect.tsx

```diff
@@ -43,9 +43,9 @@ export const SearchSelect = (props: IProps) => {
     return find(
       options,
       (v: IItem) =>
-        !isNil(v.value) &&
+        !isNil(v?.value) &&
         !isNil(value) &&
-        v.value.toString() === value.toString()
+        `${v.value}`.toString() === `${value}`.toString()
```

Comment:
This string template is not necessary the function isNil checks that it is not null or undefined